elements/pfe-navigation/src/pfe-navigation.scss

```diff
@@ -72,6 +72,24 @@ $variables: map-merge($variables, $nav-variables);
   z-index: #{pfe-zindex(navigation)};
 }
 
+:host([pfe-full-width]) {
+  .pfe-navigation {
+    &__container {
+      @each $size in ("md", "lg", "xl") {
+        @media screen and (min-width: pfe-breakpoint( #{$size} ) ) {
+          @if ($size == 'md') {
```

Comment:
I believe these should use double quotes for consistency

Reply:
I think this has been taken care of.